shared/bundle_analysis/models.py

```diff
@@ -43,11 +43,12 @@
 
```

Comment:
Add a session_id FK for Chunk and Modules, as each Chunk and Module can only belong to one session (ie stat file / bundle report). This way we can quickly navigate to the right objects of a session and it will fix an issue where Chunks/Modules from other bundles are incorrectly associated with the current bundle being processed.